lib/pkg.ml

```diff
@@ -139,13 +139,24 @@ let delegate p =
 let build_dir p =
   match p.build_dir with Some b -> Ok b | None -> Ok (Fpath.v "_build")
 
+let is_backup_file p =
```

Comment:
How about moving this function to a separate module? Sinc we want to get rid of `Pkg` as it is I'd rather avoid adding functionalities to it.

I think it would also make sense to test it!

Reply:
Agreed, I will change this.

Reply:
done